VimeoUpload.podspec

```diff
@@ -21,7 +21,9 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.9'
 
   s.source       = { :git => "https://github.com/vimeo/VimeoUpload.git", :tag => s.version.to_s }
-  s.source_files  = "VimeoUpload", "VimeoUpload/**/*.{swift}"
+  s.source_files  = "*.{h}", "VimeoUpload", "VimeoUpload/**/*.{swift}", "VimeoUpload/**/*.{h}"
```

Comment:
Please ignore all changes i've made to the Podfile. This was some late testing / tryouts to try to install the project through cocoapods (by setting the github repo param) but we never quite got it to work due to the VIMNetworking deps :) I don't know really why this appears here - i created the PR before doing these changes (i think).